src/Aspire.Hosting/Oracle/OracleDatabaseBuilderExtensions.cs

```diff
@@ -24,7 +24,7 @@ public static class OracleDatabaseBuilderExtensions
     /// <returns>A reference to the <see cref="IResourceBuilder{OracleDatabaseContainerResource}"/>.</returns>
     public static IResourceBuilder<OracleDatabaseContainerResource> AddOracleDatabaseContainer(this IDistributedApplicationBuilder builder, string name, int? port = null, string? password = null)
     {
-        password = password ?? Guid.NewGuid().ToString("N");
+        password = password ?? Guid.NewGuid().ToString("N").Substring(0, 30);
```

Comment:
We should centralize this generated password logic into `PasswordUtils` @mitchdenny

Reply:
Note that it is different than SqlServer.

https://github.com/dotnet/aspire/blob/bc220e3de6d8b398b22e0d0b6d338a8ab1ab1ba1/src/Aspire.Hosting/SqlServer/SqlServerBuilderExtensions.cs#L25

and MySql

https://github.com/dotnet/aspire/blob/bc220e3de6d8b398b22e0d0b6d338a8ab1ab1ba1/src/Aspire.Hosting/MySql/MySqlBuilderExtensions.cs#L27

Reply:
We should probably just come up with a decent password generator that we can use. What I did for SQL Server was half baked and really only appropriate for inner loop (which luckily is where it is used). I'd be happy to see this go in without change this password generation logic and instead see an issue created where we properly develop out PasswordUtils than can meet our various use cases (needs to be flexible enough to side step various limitations around what the containerized tool can used, and what we can escape in the connection string.

Reply:
I know an issue hasn't been created yet, but I saw this comment and opened a PR that might address it: https://github.com/dotnet/aspire/pull/1469.